app/templates/components/forms/session-speaker-form.hbs

```diff
@@ -318,6 +318,7 @@
                 {{/ui-dropdown}}
               {{else}}
                 {{#ui-dropdown class='search selection' onChange=(action (mut (get data.session field.fieldIdentifier))) as |execute mapper|}}
+                  {{input type='hidden' name=(if field.isRequired (concat 'session_' field.fieldIdentifier '_required') (concat 'session_' field.fieldIdentifier)) value=data.session.fieldIdentifier}}
```

Comment:
everywhere, ```(get data.session.field.fieldIndentifier)``` is used to get the value. Either it's moot to use it everywhere, or it is the reason why your changes are not reflecting.
Please check.

Reply:
Will check it, As the track field is not even getting send, As the speaker is not able to submit the session through CFS

Reply:
@CosmicCoder96 @ritikamotwani @niranjan94, I got the error, Its not due to the validations, The above-written line works well in all cases but when a user fills up `Session Types` leaving Track empty, The forms get submitted due to the relation of `Session Types` with `Track` , But their the error occurs the server returns a 422, As to specify `Session Type` a valid relation with `Track` is needed, To overcome this error, I introduced a new validation in the `Speaker-Session Form` 
```
$.fn.form.settings.rules.isTrackField = () => {
    return this.get('data.session.track');
  };
```
This rule will be applied to `Session Types` so that user cant submit form with `Track` being blank .
But when I run with the following validation I get the error `Cannot read property 'settings' of undefined`.
Can you point out what am I doing wrong here?

Reply:
@kushthedude try `window.$.fn.form.settings.rules.isTrackField`

Reply:
![image](https://user-images.githubusercontent.com/44091822/59701246-cb562d80-9212-11e9-9c49-137fc58f2f9c.png)
@niranjan94 It was of no good , Any other way to add custom validations in form ?

Reply:
@CosmicCoder96  ?

Reply:
@kushthedude where are you adding this ?

Reply:
@niranjan94 Please see https://github.com/fossasia/open-event-frontend/pull/3092/files#r295161950

Reply:
@CosmicCoder96 @niranjan94 , I added validation for `Session_Track` but I don't know as soon as I fill up `SessionType` without filling the `Session_Track`, Session track validations gets bypassed, Any help is highly appreciated